website/src/content/api-reference/deployment-previews.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Deployment previews
+---
+
+<Subtitle>
+
+Deployment previews let you test your Custom Application features before publishing them to the production environment.
```

Comment:
Nit: I would rephrase this a bit, it might seem 

```suggestion
Deployment previews allow you to preview changes to your Custom Application using a unique URL.
```

Reply:
Thanks, @emmenko. It looks clearer. Although, "allows" is a prohibited word in our styleguide. I'll rephrase to keep your intent.

<img width="563" alt="20-02-2023-at-15-21-20@2x" src="https://user-images.githubusercontent.com/28081510/220071638-ee685ed5-59d7-4652-8fa0-16bd88f1ec5c.png">